### DISCUSSION TITLE
Can I change,which configuration and dependencies every prefix will have?

### DISCUSSION THREAD

**Discussion by ghost** [bottlesdevs/Bottles#200] [Q&A] [answered]
I mean,can I set that for example mono gets installed for every new prefix automatically or that things in the wine.dfg are changed for every new prefix automatically?

**Comment by mirkobrombin** [comment, marked as answer]
Bottles v4 will support configurations backup, so you can export your bottle.json configuration and import again to create a new bottle with the same specs.
Ref: https://github.com/bottlesdevs/Bottles/issues/133
